src/Components/Navigation/Header.js

```diff
@@ -1,46 +1,161 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './Header.css';
 import {
   NavLink,
 } from 'react-router-dom';
-import logo from '../../Assets/Images/TCK PNG Logo.png';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import logo from '../../Assets/Images/TCK SVG Logo.svg';
```

Comment:
Make this upper case to match your formatting of all of the other icons!
```suggestion
import Logo from '../../Assets/Images/TCK SVG Logo.svg';
```